bodo/pandas/vendor/duckdb/src/CMakeLists.txt

```
@@ -69,10 +69,10 @@ if(AMALGAMATION_BUILD)
   target_link_libraries(duckdb_static ${DUCKDB_SYSTEM_LIBS})
   link_threads(duckdb_static)
   link_extension_libraries(duckdb_static)
-
-  install(FILES "${PROJECT_SOURCE_DIR}/src/amalgamation/duckdb.hpp"
-                "${PROJECT_SOURCE_DIR}/src/include/duckdb.h"
-          DESTINATION "${INSTALL_INCLUDE_DIR}")
+  # Bodo change: I think we removed these
```

```suggestion
  # Bodo change: remove ...
```
And remove commented lines below?

---

IMO leaving them commented out is better than removal, it makes looking at the diff easier to match up when upgrading